src/mmrelay/matrix_utils.py

```
@@ -1073,27 +1075,114 @@ def bot_command(
+        If the backend's `send_message` returns an awaitable, this function will run it to completion before returning the result.
+    """
+    try:
+        result = backend.send_message(
+            text=text,
+            channel=channel,
+            destination_id=destination_id,
+            reply_to_id=reply_to_id,
+        )
+        if inspect.isawaitable(result):
+            return asyncio.run(cast(Coroutine[Any, Any, Any], result))
+        return result
```

Ruff (0.14.14)

Consider moving this statement to an `else` block

(TRY300)